src/layouts/ProfileViewLayout/ProfileViewLayout.jsx

```diff
@@ -1,20 +1,35 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { useBackend } from "../../state/use-backend";
```

Comment:
Layouts should not use stateful hooks.
Let's try to keep Layouts and Components as dumb as possible.

I think it is better to flow data as properties.